### DISCUSSION TITLE
Resize element with element tools control

### DISCUSSION THREAD

**Discussion by malomiquel** [clientIO/joint#1888] [Q&A] [answered]
Hello,

I have a problem with the element tool control.
I would like to resize an element in both directions, right and left. But when I try to resize my element to the left nothing happens.

Here is my code:

![code](https://user-images.githubusercontent.com/83480447/201648001-c04d080e-206a-4546-a9c3-be380e047ac9.png)

<img width="290" alt="Capture d’écran 2022-11-14 122251" src="https://user-images.githubusercontent.com/83480447/201648019-20961111-554e-43d5-9670-bf3955e03282.png">

Thanks

**Comment by kumilingus** [comment, marked as answer]
If you resize the element to the left, you change not only its `width`, but also the `x` coordinate.

You need to change the `position` too.
```js
el.position(newX, currentY);
```

There is also a built-in way to change both `width` and `x` using the [resize()](https://resources.jointjs.com/docs/jointjs/v3.6/joint.html#dia.Element.prototype.resize) method.

```js
el.resize(newWidth, currentHeight, { direction: 'left' })
```

**Reply by malomiquel** [reply]
Thank you for your answer but is there a solution to do just with the element tool Control ?

**Reply by kumilingus** [reply]
Yes. Please see [solution](https://codepen.io/jointjs/pen/ZERyVxe/c92472ae00ccfc127b934c6b693d3e15).
